tensorflow/core/kernels/fused_batch_norm_op.cc

```diff
@@ -34,6 +34,7 @@ limitations under the License.
 #include "tensorflow/core/framework/register_types.h"
 #include "tensorflow/core/framework/tensor.h"
 #include "tensorflow/core/framework/tensor_types.h"
+#include "tensorflow/core/kernels/gpu_utils.h"
```

Comment:
You need to add a bazel dep to include this header.

Reply:
indeed, I missed a couple of places which caused cpu build brake, thanks